src/Calculator/include/calculator.h

```diff
@@ -0,0 +1,30 @@
+﻿#pragma once
+
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+class Calculator{
+
+public:
+    Calculator();
+
+    double calculate(std::string input);
+    std::string error() const;
+
+private:
+
```

Comment:
Think about an idea that most of function should be cons be default.
Check if some of you fucntions may not change the sate of the class, but just return value.

Reply:
Thanks, done